server/views/components/button-link/index.config.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "button-link",
+  "handle": "button-link",
+  "context": {
+    "href": "wellcomecollection.org",
+    "text": "this is a button"
+  },
+    "variants": [
+        {
+            "name": "with icon",
+            "context": {
+                "icon": "download-1",
+                "text": "this is a link with an icon"
+            }
+        },
+        {
+            "name": "light button",
+            "context": {
+                "modifierClasses": "btn--light",
+                "text": "this is a link"
+            }
+        },
+        {
+            "preview": "@preview-dark",
+            "name": "dark button with icon",
+            "context": {
+                "modifierClasses": "btn--dark",
```

Comment:
It might be nice to start making a list (github issue?) of things like this, as we've had a few, and I'm starting to forget them.